tests/utils.py

```diff
@@ -76,32 +78,27 @@ def internal_sideeffect(*args, **kwargs):
     if not answer:
         # Means it is not in cache.
         # So, execute the real function and store in cache
-        answer = alternative(*args)
+        answer = real_json_query(*args)
         RES[key] = answer
     return answer
 
 
-def dastollervey_datasaver(func):
+def qrexec_datasaver(func):
     "This is the decorator to save qrexec call data"
 
-    def wrapper(*args, **kwargs):
+    def wrapper(self, *args, **kwargs):
         global CALLNUMBER
         global RES
         # This is the filename to store the results
         filename = os.path.join("data", func.__name__ + ".json")
-        # if the logout.txt file is there, means we have a logout scenario
```

Comment:
was this never necessary?

Reply:
discussed out-of-band some ideas about making tests less brittle so that we can run them in random order in the future. for now, since the logout test will be the last test to run, this code is fine to remove